_security-analytics/usage/correlation-graph.md

```diff
@@ -0,0 +1,50 @@
+{: .warning }
+
+The correlation graph is a security findings knowledge graph. It provides a visualization for information generated by the correlation engine and allows you to focus on specific correlations and inspect them in greater detail. Information in the graph includes findings by log type, the severity levels for the findings, the correlations drawn between findings, and the relevance of the correlations, among other details. You can also manipulate the graph to gain further insights into specific events of interest. This includes filtering findings by date and time, zooming in on the relationship between specific findings and their correlations, and filtering by log type and severity level. Use this section to learn more about using the graph. 
+
+## Acccessing the graph
+
+Begin by selecting **Security Analytics** in the Dashboards home page menu. Then select **Correlations** from the Security Analytics menu on the left side of the screen. The **Correlations** page is displayed.
+
+<img src="{{site.url}}{{site.baseurl}}/images/Security/sec-analytics/corr-graph.png" alt="The correlation graph" width="85%">
+
+## Interpreting the graph
+
+The graph displays findings as nodes with colored borders expressing their severity level. A three-letter abbreviation inside the node indicates the log type. The lines that connect the findings represent the correlations between them. A heavy line indicates a strong correlation, while a light line shows a weaker connection.
+
+<img src="{{site.url}}{{site.baseurl}}/images/Security/sec-analytics/corr-graph-detail.png" alt="The correlation graph" width="40%">
+
+## Using the graph
+
+You can control which findings are displayed in the graph by filtering by severity level, log type, and time filter. The time filter controls the findings that appear in the graph by limiting the dates on which the findings were generated.
+* Use the **Severity** dropdown list to select which findings appear in the graph by their severity level. The number beside the list name shows how many severity levels are being shown in the graph.
+* Use the **Log types** dropdown list to select which log types to show in the graph. The number beside the list name shows how many log types are being shown in the graph.
+* Select **Reset filters** to return the dropdown lists to their default, showing all items.
+* Use the time filter to set the the date range and show only those findings that were generated within that time span. Select **Refresh** to bring the current number of findings up to date.
+
+You can drill down in the graph to look at correlations associated with a specific finding by selecting the finding in the graph. The graph then changes to show only the selected finding along with the constellation of findings correlated to it.
```

Comment:
Let's use something other than "drill down," which may not be understood by all readers.

Reply:
Yes.
"You can focus on a particular area of the graph to look at correlations associated with a specific finding..."